iceoryx_posh/source/mepoo/memory_manager.cpp

```diff
@@ -58,7 +58,7 @@ void MemoryManager::addMemPool(BumpAllocator& managementAllocator,
                 return log;
             } << "These mempools must be added in an increasing chunk size ordering. The newly added  MemPool [ "
                  "ChunkSize = "
-              << adjustedChunkSize << ", ChunkPayloadSize = " << static_cast<uint32_t>(chunkPayloadSize)
+              << adjustedChunkSize << ", ChunkPayloadSize = " << static_cast<uint64_t>(chunkPayloadSize)
               << ", ChunkCount = " << static_cast<uint32_t>(numberOfChunks) << "] breaks that requirement!");
```

Comment:
Small nitpick. The casts can be removed. There once was a bug in the `LogStream` implementation which was fixed a while back.

No hard requirement for a merge since this was already there, just a nice to have :)